project/Dependencies.scala

```diff
@@ -143,6 +143,15 @@ object Dependencies {
   val pact4sCirce =       "io.github.jbwheatley"  %% "pact4s-circe"     % pact4sV
   val okHttp =            "com.squareup.okhttp3"  % "okhttp"            % "4.11.0"
 
+  val workSpaceManagerV = "0.254.782-SNAPSHOT"
```

Comment:
`Dependencies.scala` and `Merging.scala` changes also copied from https://github.com/DataBiosphere/leonardo/pull/3401